### PR TITLE
Add Drone CI support

### PR DESCRIPTION
**Pull request by gabro**
Almost identical to Travis CI, I just replaced variable names with the ones from http://readme.drone.io/usage/variables/

### CODE REVIEW

**Comment by orta** [comment]
👍

**Comment by gabro** [comment]
finally! OSX being case-insensitive didn't really help...

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ## Master
 
 * Considerable under-the-hood changes around the DSL, shouldn't affect end-user Dangerfiles though - orta
+* Add support for Drone CI
```
Comment (jeroenvisser101) [inline CHANGELOG.md]: Could you add your name here? Thanks!

**Comment by jeroenvisser101** [comment]
Aside from the super tiny changelog change, this LGTM! 🚀

**Comment by jeroenvisser101** [comment]
Also, thanks for contributing to Danger!

**Comment by gabro** [comment]
Changelog edit done, thank you for the great tool!

**Pull request merged**